maestro-test/src/test/resources/118_scroll_until_visible_negative.yaml

```diff
@@ -0,0 +1,8 @@
+appId: com.example.app
+---
+- scrollUntilVisible:
+    element:
+      id: "maestro"
+    direction: DOWN
+    speed: 110
```

Comment:
Setting speed > 100 yields a negative duration so this is intended

Reply:
Hmmm, might be some change that we have setting on duration for this maestro release.

I believe this one: 

https://github.com/mobile-dev-inc/maestro/pull/1997/files

Reply:
Yes that is suspicious but I couldn't figure out how this would yield a negative duration _without_ setting the speed in a flow.